nyholm/psr7/0.3/config/packages/nyholm_psr7.yaml

```diff
@@ -0,0 +1,31 @@
+services:
+    _defaults:
+        public: false
+
+    ## Register nyholm/psr7 services for autowiring with Interop\Http
+    #Interop\Http\Factory\RequestFactoryInterface: '@nyholm.psr7.message_factory'
+    #Interop\Http\Factory\ResponseFactoryInterface: '@nyholm.psr7.message_factory'
+    #Interop\Http\Factory\ServerRequestFactoryInterface: '@nyholm.psr7.server_request_factory'
+    #Interop\Http\Factory\StreamFactoryInterface: '@nyholm.psr7.stream_factory'
+    #Interop\Http\Factory\UploadedFileFactoryInterface: '@nyholm.psr7.uploaded_file_factory'
+    #Interop\Http\Factory\UriFactoryInterface: '@nyholm.psr7.uri_factory'
+
+    ## Register nyholm/psr7 services for autowiring with HTTPlug
+    #Http\Message\MessageFactory: '@nyholm.psr7.message_factory'
```

Comment:
Wouldn't it make sense to add aliases for `RequestFactory` and `ResponseFactory` too?

Reply:
Yeah, you might be correct. 
Thank you.